File: app/src/main/java/skripsi/kohonen/signaturerecognition/TandaTangan.java
package skripsi.kohonen.signaturerecognition;

/**
 * Created by fep on 29/03/2017.
 */

public class TandaTangan {
    private int id;
    private String nama;
    private String nilai;

    public TandaTangan() {
    }
    public TandaTangan(int id, String nama, String nilai) {
        this.id = id;
        this.nama = nama;
        this.nilai = nilai;
    }

    public static String arrToStr(byte[] arr) {
        String str = "";
        for (int i=0; i<arr.length; i++) {
            str += arr[i];
        }
        return str;
    }

    public static byte[] strToArr(String str) {
        byte[] arr = new byte[str.length()];
        for (int i=0; i<arr.length; i++) {
            arr[i] = (byte) Character.getNumericValue(str.charAt(i));
        }
        return arr;
    }

    public void setId(int id) {
        this.id = id;
    }

        public void setnama(String nama) {
            this.nama = nama;
        }
        public void setNilai(String nilai) {
            this.nilai = nilai;
        }

    public int getId() {
        return id;
    }
    public String getNilai() {
        return nilai;
    }
    public String getnama() {
        return nama;
    }
}

File: app/src/main/java/skripsi/kohonen/signaturerecognition/TandaTangan.kt
package skripsi.kohonen.signaturerecognition

/**
 * Created by fep on 29/03/2017.
 */
class TandaTangan {
    private var id = 0
    private var nama: String? = null
    private var nilai: String? = null

    constructor() {}
    constructor(id: Int, nama: String?, nilai: String?) {
        this.id = id
        this.nama = nama
        this.nilai = nilai
    }

    fun setId(id: Int) {
        this.id = id
    }

    fun setnama(nama: String?) {
        this.nama = nama
    }

    fun setNilai(nilai: String?) {
        this.nilai = nilai
    }

    fun getId(): Int {
        return id
    }

    fun getNilai(): String? {
        return nilai
    }

    fun getnama(): String? {
        return nama
    }

    companion object {
        fun arrToStr(arr: ByteArray): String {
            var str = ""
            for (i in arr.indices) {
                str += arr[i]
            }
            return str
        }

        fun strToArr(str: String?): ByteArray {
            val arr = ByteArray(str!!.length)
            for (i in arr.indices) {
                arr[i] = Character.getNumericValue(str[i]).toByte()
            }
            return arr
        }
    }
}